app/src/main/java/com/nextcloud/talk/activities/CallActivity.java

```diff
@@ -2591,12 +2593,16 @@ private void playCallingSound() {
 
     private void stopCallingSound() {
         if (mediaPlayer != null) {
-            if (mediaPlayer.isPlaying()) {
-                mediaPlayer.stop();
-            }
+            try {
+                if (mediaPlayer.isPlaying()) {
+                    mediaPlayer.stop();
+                }
 
-            mediaPlayer.release();
-            mediaPlayer = null;
+                mediaPlayer.release();
```

Comment:
if mediaPlayer.stop() fails, then it will never be released.
Put it in finally, so that it will always be reached.